proto/dydxprotocol/revshare/query.proto

```diff
@@ -1,7 +1,23 @@
 syntax = "proto3";
 package dydxprotocol.revshare;
 
+import "gogoproto/gogo.proto";
+import "dydxprotocol/revshare/params.proto";
+
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/revshare/types";
 
 // Query defines the gRPC querier service.
-service Query {}
\ No newline at end of file
+service Query {
+  // MarketMapperRevenueShareParams queries the revenue share params for the
+  // market mapper
+  rpc MarketMapperRevenueShareParams(QueryMarketMapperRevenueShareParams)
+      returns (QueryMarketMapperRevenueShareParamsResponse);
```

Comment:
nit: can you add an http api annotation like this [example](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/prices/query.proto#L16)?